keps/sig-node/4622-topologymanager-max-allowable-numa-nodes/README.md

```diff
@@ -234,10 +233,6 @@ For beta:
 
 - Verify the input validation with the existing e2e tests(e.g. 9 or 10 or something bigger than the current default but not "too big")
 
-For GA:
-
-- degrading the node and checking the node is reported as degraded
-
```

Comment:
why was this removed?

Reply:
I kinda don't remember the context here, do we need this e2e to test it?

Reply:
that was about making the state of the feature observable to others. We already has issues reported in this area: https://github.com/kubernetes/kubernetes/issues/131738

With the new sig-arch requirements effective 1.34, if we do this for GA we need to have another beta